symfit/contrib/interactive_guess/interactive_guess.py

```diff
@@ -8,53 +8,71 @@
 
 import matplotlib.pyplot as plt
 import numpy as np
+from scipy.stats import gaussian_kde
 
 plt.ioff()
 
 #from pkg_resources import parse_version
 
-#SUPPORTED_VERSION = '0.3.0'
+#SUPPORTED_VERSION = '0.3.6'
```

Comment:
I'm not sure this is actually all that relevant, since interactive_guess is shipped with symfit anyway. So it should always support the version it is packaged with, and using it with a different version of symfit would require a lot of hacking, and if a user does that they deserve whatever happens to them.